js/gl.ts

```diff
@@ -812,10 +788,33 @@ export class VideoModesGL implements VideoModes {
         return blitted;
     }
 
+    buildScreen(mainData: ImageData, mixData?: ImageData | null) {
```

Comment:
There isn't much difference with the other implementation. Could this inherit from the other or something?

Reply:
Yes, it seems likely. I hope to pull common code into the base class in the next refactoring phase.